### ISSUE TITLE
Issue with SupportMapFragment

### ISSUE THREAD

**Issue by psgganesh** [priyaaank/bloodtorrent#16]
Hi Priyaank, thanks for such a wonderful project, thank you for making it open, I am trying to install and configure it on my local. I am done with configuring catarava, ruby, nodejs and android sdk and adt. I have cloned up number picker and google play services into LibraryProjects and imported them to this project. 

I have stumbled upon this last error. SupportMapFragment navigates me to configure build path. Any clues? Am not sure if you have encountered the same.

**Comment by priyaaank** [comment]
Hey @psgganesh, Thanks for checking out this project. I haven't looked at error by checking out this project again. However have you configured project to use Android support library? 
http://developer.android.com/tools/support-library/index.html

If you can share what IDE is it that you are using and how you have configured project I might be able to help better.

**Comment by psgganesh** [comment]
Perfect, that worked, i further added Rhino, js, jackson mapper; but am still stuck with one last error, which mentioned the following

The type org.codehaus.jackson.JsonParser cannot be resolved. It is indirectly referenced from required .class files

not sure how to go about it, any clues? am so excited to get this started :)

Added to this i use Eclipse IDE, Ubuntu 12.04

**Comment by psgganesh** [comment]
Hi @priyaaank ; it was not possible, below is my project setup

![image](https://f.cloud.github.com/assets/2221525/2279548/1dad1f4c-9f72-11e3-858f-351520ed679c.png)

Please let me know if more information is required; @priyaaank thanks again for taking your time helping all us out...

**Comment by priyaaank** [comment]
@psgganesh Hey, sorry I forgot to respond earlier. 

So I looked back at the configuration and wanted to point out that Calatrava uses ivy for java dependency management. You can find the ivy file here[https://github.com/priyaaank/bloodtorrent/blob/master/droid/b/ivy/ivy.xml]

Running bootstrap rake tasks would typically pull down the dependencies as well. However in worst case you could run the ivy task directly and pull all the dependencies. This should pull four dependencies:
- Rhino
- Jackson
- Android Support Lib
- Common-Lang3

Once these dependencies are there, you should be able to build the app and get up and running. What are the commands you have run to set the project up. You would need to do at least the following:
- bundle install
- bundle exec rake bootstrap
- bundle exec rake droid:deploy

If you get error on command line while building the app, I would recommend building the android app from eclipse. (or IntelliJ Idea). I can definitely help, if you have any more errors. Let me know.

**Comment by psgganesh** [comment]
Tried everything again installed ruby and dependencies from scratch, i thought it would be done and i can close the issue... but looks like it's somehow still stuck... :(

**Comment by priyaaank** [comment]
When you have some time ping me on priyaaank @ gmail. Lets chat, I am happy to do a screen share and try to resolve your problem. I haven't set it up in a while on Ubuntu, but I would have assumed that it would be easy to setup.

**Comment by psgganesh** [comment]
Yup sure, will buzz you once am online.. thank you for all the help...
